backend/Controllers/SightingController.cs

```diff
@@ -18,10 +18,17 @@ ISightingService sightings
         }
         
         [HttpGet("/approved")] 
-        public IActionResult<ListResponse<Sighting>>GetApprovedSightings()
+        public ActionResult<ListResponse<Sighting>>GetApprovedSightings()
         {
             var approvedSightings = _sightings.GetApprovedSightings();
-            return new ListResponse<Sightings>(approvedSightings);
+            return new ListResponse<Sighting>(approvedSightings);
+        }
+
+        [HttpGet("/unconfirmed")] 
+        public ActionResult<ListResponse<Sighting>>GetUnconfirmedSightings()
+        {
+            var unconfirmedSightings = _sightings.GetUnconfirmedSightings();
+            return new ListResponse<Sighting>(unconfirmedSightings);
```

Comment:
We're using the word "pending" for these
```suggestion
        [HttpGet("/pending")] 
        public ActionResult<ListResponse<Sighting>>GetPendingSightings()
        {
            var pendingSightings = _sightings.GetPendingSightings();
            return new ListResponse<Sighting>(pendingSightings);
```